chapters/usage-profile.md

```diff
@@ -0,0 +1,326 @@
+
+Using `xml:base`:
+
+```text
+<rdf:RDF xml:base="http://acme.com/spdxdocs/acmeproj/v1.2/1BE2A4FF-5F1A-48D3-8483-28A9B0349A1B">
+    ...
+    <UsageScope rdf:about="#SPDXRef-1">
+    ...
+    </UsageScope>
+</rdf:RDF>
+```
+
+Using document URI:
+
+```text
+<UsageScope rdf:about="http://acme.com/spdxdocs/acmeproj/v1.2/1BE2A4FF-5F1A-48D3-8483-28A9B0349A1B#SPDXRef-1">
+    ...
+</UsageScope>
+```
+
+## 31.4 Package Release Date field <a name="31.4"></a>
+
+### 31.4.1 Description
+
+This field provides a place for recording package release date
+
+Table 31.4 — Metadata for the package release date
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | `YYYY-MM-DDThh:mm:ssZ`<br>where:<br><ul><li>`YYYY` is year</li><li>`MM` is month with leading zero</li><li>`DD` is day with leading zero</li><li>`T` is delimiter for time</li><li>`hh` is hours with leading zero in 24 hour time</li><li>`mm` is minutes with leading zero</li><li>`ss` is seconds with leading zero</li><li>`Z` is universal time indicator</li></ul> |
+
+
+### 31.4.2 Intent
+
+Description of the package release date for strict identification of the prerequisite assumptions of usage scope by the package supplier.
+
+### 31.4.3 Examples
+
+EXAMPLE 1 Tag: `PackageReleaseDate:`
+
+```text
+PackageReleaseDate: 2010-01-29T18:30:22Z
+```
+
+EXAMPLE 2 RDF: Property `spdx:packageReleseDate` in class `spdx:UsageScope`
+
+```text
+<UsageScope rdf:about="...">
+    <packageReleaseDate> 2010-01-29T18:30:22Z </packageReleaseDate>
+</UsageScope>
+```
+
+## 31.5 Package Built Date field <a name="31.5"></a>
+
+### 31.5.1 Description
+
+This field provides a place for recording built date of the package 
+
+Table 31.5 — Metadata for the package built date
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | `YYYY-MM-DDThh:mm:ssZ`<br>where:<br><ul><li>`YYYY` is year</li><li>`MM` is month with leading zero</li><li>`DD` is day with leading zero</li><li>`T` is delimiter for time</li><li>`hh` is hours with leading zero in 24 hour time</li><li>`mm` is minutes with leading zero</li><li>`ss` is seconds with leading zero</li><li>`Z` is universal time indicator</li></ul> |
+
+
+### 31.5.2 Intent
+
+Description of the built date of the package for strict identification of the prerequisite assumptions of usage scope by the package supplier.
+It should recorded from build system tools or date field of the package file.
+
+### 31.5.3 Examples
+
+EXAMPLE 1 Tag: `PackageBuiltDate:`
+
+```text
+PackageBuiltDate: 2010-01-29T18:30:22Z
+```
+
+EXAMPLE 2 RDF: Property `spdx:packageBuiltDate` in class `spdx:UsageScope`
+
+```text
+<UsageScope rdf:about="...">
+    <packageBuiltDate> 2010-01-29T18:30:22Z </packageBuiltDate>
+</UsageScope>
+```
+
+## 31.6 Valid Until Date field <a name="31.6"></a>
+
+### 31.6.1 Description
+
+Identify expiration date of designated "usage scope" descriptions which declared by supplier of the package. 
```

Comment:
I'm not sure I understand what an "expiration date" or "valid until" means in this context.

Is this meant to say that the actual item being described by the SPDX element (e.g. the actual software package) is "expired" or "invalid" after this date? Or is this saying that the SPDX metadata itself (e.g. the metadata in the SPDX document) is "expired" or "invalid" after this date?

Reply:
It is referring to the period of support.   Example is the kernel LTS is supported by the community until a specific date.   Automated checks can then monitor when these dates are known.